crypto/lhash/lhash.c

```diff
@@ -166,18 +189,24 @@ void *OPENSSL_LH_retrieve(OPENSSL_LHASH *lh, const void *data)
 {
     unsigned long hash;
     OPENSSL_LH_NODE **rn;
-    void *ret;
+    void *ret = NULL;
```

Comment:
when we return NULL because of an internal error, that is
probably worse than wrong statistic counters.  Usually that
will result in adding the same key twice, which makes the
hash inconsistent.
I would like a do_tsan variable better as it is done in getrn below

Reply:
I've found a better way :)